src/main/webapp/data/userMapData.json

```diff
@@ -1238,6 +1266,7 @@
       "Kit Carson World School",
       "Knox High School",
       "Lakeridge Junior High",
+      "Lake Sumter State College",
```

Comment:
This should come before `Lakeridge Junior High` - shorter words should come before longer words in alphabetical ordering.

Reply:
Thanks for correcting. Changed accordingly.